src/ppp_connection_manager.sh

```diff
@@ -4,10 +4,12 @@ source functions.sh
 
 for i in {1..4}; do
     bash configure_modem.sh
+    MODEM_CONFIG=$?
```

Comment:
MODEM_CONFIG is set here to exit code of configure_modem.sh

Reply:
yes, this line is required to be added